recipes/ndsplines/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ndsplines" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ndsplines-{{ version }}.tar.gz
+  sha256: 39f1b7f0d7e0f99bc45099b87716abd62d5e5b07820680e1adb8d9188ef352b9
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools >=45
+    - setuptools-scm
+    - cython
+    - numpy  # [py<312]
+    - numpy >=1.26.0  # [py>=312]
+    - pip
+  run:
+    - python
+    - scipy
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - ndsplines
+  commands:
```

Comment:
As an additional test command, something like this might be useful to catch any problems loading the (default) compiled backend:

```
- python -c "import ndsplines; ndsplines.get_impl() == 'cython'"
```

Reply:
Thanks, I added the test in https://github.com/conda-forge/staged-recipes/pull/24743/commits/59295932fd510536c5b624659d9a35beb6a530d2.